### DISCUSSION TITLE
Adding links to a chart

### DISCUSSION THREAD

**Discussion by andrewducker** [EvotecIT/PSWriteHTML#463] [Q&A]
My chart columns have explanations.  I'd like to make the Chart Bar name into a link, but putting in the result of a New-HTMLText doesn't work there, and neither "Chart" nor "ChartBar" have the InvokeHTMLTags parameter.

Failing that, I'd settle for putting in some links after the chart - but that just puts them to the right of the chart, even if I add in a "BR".  How would I ensure that they go underneath the chart rather than to the right?

Thank you!

**Comment by tomerikko** [comment]
i didnt tried on the columns but I did added hyperlinks on the content of the columns.

what I did is I made sure I write the HREF (google) structure into the field before I send the data into the PSwriteHTML module, and then the table that is built comes with the hyperlinks in it.  very useful.
